start/routes.ts

```diff
@@ -0,0 +1,26 @@
+
+import Route from '@ioc:Adonis/Core/Route'
+
+Route.get('/', async () => {
+  return "Hello Adonis!"
+})
+
+Route.group(()=>{
+
+  Route.group(()=>{
+    // Route.get("/user/profile","ProfileController.view")
+    Route.get("/user/profile","ProfileController.showProfile")
+    Route.post("/user/profile","ProfileController.create")
+    Route.put("/user/profile","ProfileController.update")
+    Route.delete("/user/profile","ProfileController.delete")
```

Comment:
as per requirement user has to  be deleted by mobile number, you can pass mobile in params to delete profile